frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.js

```diff
@@ -186,8 +187,7 @@ function StartingJobModal(props: StartingJoblModalProps) {
       <br />
       <div style={{ textAlign: "center" }}>
         <img
-          //TODO: get an image
-          src="/assets/images/nuclei_inferral_example.jpg"
+          src={`/assets/images/${_.snakeCase(jobName)}_example.jpg`}
```

Comment:
this will make it very hard to search where the file is used. I'd suggest to create a mapping object from jobName to path to make it easier to find. something like this:

```
const jobNameToImagePath = {
  "neuron_inferral": "neuron_inferral_example.jpg",
  ...
}
```

Reply:
also: the screenshot quality of the new image is quite poor. maybe set the zoom value to 1 to avoid resizing issues? also, I'd hide the "crosshair" in the image.

Reply:
Thanks for the feedback. I did notice that the image had such a bad resolution. It should be better blazingly sharp 🗡️  now.